staff/noli-pascual/semantic-blog/article.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>semantic article</title>
+    <link rel="stylesheet" href="style1.css">
+</head>
+<body>
+    <header>
+        <figure>
+            <img src="skylab-logo.png" id ="logo" alt="logo-skylab">
```

Comment:
whenever possible, avoid using `id` to relate styles and elements. it  is preferable to use `class`. so, remove all the `id` and use `class` instead. same applies for the other elements.